README.md

```diff
@@ -109,6 +125,18 @@ The following options are available:
      --marketplace-address  Address of deployed Marketplace contract.
      --validator            Enables validator, requires an Ethereum node [=false].
      --validator-max-slots  Maximum number of slots that the validator monitors [=1000].
+                            If set to 0, the validator will not limit the maximum number of slots it
+                            monitors.
+     --validator-groups     Slot validation groups [=ValidationGroups.none].
+                            A number indicating total number of groups into which the whole slot id space
+                            will be divided. The value must be in the range [2, 65535]. If not provided, the
+                            validator will observe the whole slot id space and the value of the
+                            --validator-group-index parameter will be ignored. Powers of twos are advised
+                            for even distribution.
+     --validator-group-index  Slot validation group index [=0].
+                            The value provided must be in the range [0, validatorGroups). Ignored when
+                            --validator-groupsis not provided. Only slot ids satisfying condition [(slotId
```

Comment:
```suggestion
                            --validator-groups is not provided. Only slot ids satisfying condition [(slotId
```

Reply:
I see I have the same typo is in the CLI. Fixed.